DiffPlex/DiffPlex.csproj

```diff
@@ -1,32 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\NuGet.props" />
   <PropertyGroup>
-    <TargetFrameworks>net35;net40;netstandard1.0</TargetFrameworks>
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <TargetFrameworks>net35;net40;netstandard1.0;netstandard2.0</TargetFrameworks>
```

Comment:
Should we drop the netstandard1. 0 target based on the link you provided?

Reply:
It says _avoid_, not don't. Targeting .NET Standard 1.0 makes DiffPlex compatible with older platforms like .NET Core 1.x.

DiffPlex doesn't use a class that doesn't support .NET Standard 1.0 at the moment, so keeping it doesn't hurt anyone.